banners/desktop/WMDE_FR_2025_Desktop_DE_01/styles/Banner.scss

```diff
@@ -0,0 +1,22 @@
+@use 'src/themes/Svingle/variables/fonts';
+
+.wmde-banner {
+	&-wrapper {
+		font-size: 14px;
+		font-family: fonts.$ui;
+		box-shadow: 0 3px 0.6em rgb( 60 60 60 / 40% );
+		background-color: var( --main-background );
```

Comment:
This background colour should actually be on the `.wmde-banner-message` element, and is wrong on the English banner too. You can see in the screenshot that the background is extending beyond the red-bordered box.

You can use the same css variable but apply it in `src/themes/Treedip/Message/Message.scss`

![Uploading Screenshot 2025-07-01 at 12.28.55.png…]()

Reply:
I applied the changes you suggested. I have still kept the background color in `Banner.scss`, because without that the banner looked like following:
![Screenshot from 2025-07-01 15-03-43](https://github.com/user-attachments/assets/601078a2-dd4e-45a9-bc29-7594a415c7d0)